arch/arm/boot/dts/imxrt1170.dtsi

```diff
@@ -360,5 +362,16 @@
 			/* TBD */
 			status = "disabled";
 		};
+
+		adc1: adc@40050000 {
+			/* TBD */
```

Comment:
Why TBD's in those two sections? Shouldn't we add a more meaningful comment instead?